src/inverseRobot.js

```diff
@@ -1,30 +1,22 @@
 'use strict';
 
-/**
- * Situs inversus
- * Transposition of internal organs is a variant of internal anatomy, when
- * the organs are located in a mirror image.
- * Something similar happened to our robot Kolli. His keys became values,
- * and values became keys. Help to repair the robot. Create a 'inverseRobot'
- * function that takes 'robot' as a parameter and returns a new object in which
- * keys will change places with values.
- *
- * If any of the object values are repeated, return 'null'.
- *
- * Example:
- *
- * const kolli = { Kolli: 'name', 123: 'chipVer', 3: 'wheels' };
- * const robert = { Robert: 'name', 123: 'chipVer', 113: 'chipVer' };
- * inverseRobot(robert) === null
- * inverseRobot(kolli) === { name: 'Kolli', chipVer: '123', wheels: '3' }
- *
- *
- * @param {object} robot
- *
- * @return {object}
- */
 function inverseRobot(robot) {
-  // write code here
+  let counter = 1;
+
+  for (const key of Object.keys(robot)) {
+    const arr = Object.values(robot);
+
+    for (let i = counter; i < arr.length; i++) {
+      if (robot[key] === arr[i]) {
+        return null;
+      }
```

Comment:
You can use `hasOwnProperty` here, then get rid of `let counter`, and the second `for`